tests/cython/bindlink/test_bindlink.py

```diff
@@ -166,5 +166,26 @@ def test_execute_atom_no_return_value(self):
                         ConceptNode("deleteme")))
         self.assertEquals(result, None)
 
+
+    def test_tmp_atomspace(self):
+        ListLink(ConceptNode("foo"), ConceptNode("bar"))
+
+        get = GetLink(VariableNode("x"),
+                AndLink(
+                    PresentLink(ListLink (ConceptNode("foo"),
+                                          (VariableNode("x")))),
+                    EvaluationLink(GroundedPredicateNode("py: test_functions.func_one"),
+                        ListLink(VariableNode("x"))),
+                   EvaluationLink(GroundedPredicateNode( "py: test_functions.func_two"),
+                   ListLink (VariableNode ("x")))))
+        result = execute_atom(self.atomspace, get)
+        self.assertFalse(result.out)
+        self.assertFalse('barleycorn' in str(self.atomspace.get_atoms_by_type(types.ConceptNode)))
+        test_functions.func_two_result = TruthValue(1,1)
+        result = execute_atom(self.atomspace, get)
+        self.assertTrue(result.out)
+        # still should not be in the current namespace
+        self.assertFalse('barleycorn' in str(self.atomspace.get_atoms_by_type(types.ConceptNode)))
```

Comment:
It would be useful to have a pair of functions:
```
def is_node_in_atomspace(atom_type, str): 
    return true if ... else return false 

def is_link_in_atomspace(atom_type, atom_lsit):
      return true if ... else return false 
```

I have these in scheme, hey are almost never needed, except .. sometimes they are. Typically, to get the truth value on something, but only if that something actually exists. 

in the ideal implementation, these call into C, so that it's fast, instead of trying to search a huge list (in the real world, `get_atom_by_type` might return millions of atoms)

Reply:
I added new methods..

Reply:
Thanks!  Looks good.